src/index.js

```diff
@@ -1,6 +1,10 @@
 import './style.css';
 import dynamicScore from './modules/scores.js';
+import sound from './audio/scary.mp3';
```

Comment:
- [ ] [Optional] [Highly Recommended] the Webpack compiled with a warning that the sound file that you added is too big and causes slow loading for the app, So please change it with a small size file or delete it.